md/deploy-bundle.md

```diff
@@ -17,42 +11,34 @@ Remember that the recommended way of installing Bonita is to use the provided [T
 It saves you from doing all the following configuration at hand, as the setup tool included handles it automatically.
 :::
 
-
-## Deploy bundle content
-
-* `Tomcat-`_`version`_: a folder/file structure to be merged with an existing setup of Apache Tomcat, in order to install Bonita Portal and Bonita Engine.
-* `Wildfly-`_`version`_: a folder/file structure to be merged with an existing setup of WildFly in order to install Bonita Portal and Bonita Engine.
-<a id="platform_setup_tool" />
-* `setup`: a command-line tool that creates the Bonita Platform before it can be run. It creates the Bonita Platform internal database, and stores the runtime configuration.
-It is useful to update the configuration, locally or from a remote computer.
-* `Request_key_utils-`_`key_utils.version`_: include script files to generate license request keys.
-* `BonitaSubscription`-_`LDAPSync.version`_-`LDAP-Synchronizer` : LDAP synchronizer to synchronize your organization in Bonita with your LDAP
-* `cas-`_`cas.version`_`-module`: Module files and description to enable CAS dependency to bonita EAR.
-* `License`: license files that apply to Bonita components.
-* `README.TXT`: See this file for more details about the `deploy.zip` contents and structure. 
-
 ## Installation
 
-1. Download the deploy.zip file from the [Bonitasoft download page](http://www.bonitasoft.com/downloads-v2) for the Community edition 
+1. Download the Tomcat bundle file from the [Bonitasoft download page](http://www.bonitasoft.com/downloads-v2) for the Community edition 
 or from the [Customer Portal](https://customer.bonitasoft.com/download/request) for Subscription editions.
-2. Unzip the `deploy.zip` into a temporary folder (DEPLOY_ZIP_HOME).
-4. Follow the [Tomcat bundle steps](#tomcat-installation) or [WildFly bundle steps](#wildfly-installation)
+2. Unzip it into a temporary folder (BUNDLE_DIRECTORY).
+3. Follow the [Tomcat bundle steps](#tomcat-installation).
 
 <a id="tomcat-installation" />
 
 ## Tomcat installation
 ### Install Bonita Platform in Tomcat
 
-Copy all files and directories from DEPLOY_ZIP_HOME/Tomcat-8.5.34/server to your Tomcat root directory (TOMCAT_HOME).
+Copy, from the bundle into your Tomcat server:
+* `server/bin/setenv.sh` or `server/bin/setenv.bat`, depending if you are on Unix or Windows, to `bin`
+* The entirety of the bundle `server/conf` folder to `conf`
+* The entirety of the bundle `server/lib/bonita` folder to `lib/bonita`
+* The `server/webapps/bonita.war` and `webapps/ROOT/favicon.ico` to `webapps` and `webapps/ROOT/`
+* The entirety of the `setup` directory at the root of your tomcat server.
+* The entirety of the `server/Request_key_utils-`. Includes script files to generate license request keys.
```

Comment:
Will not be needed anymore
```suggestion
```

Reply:
Why? Request_key_utils- wont be in the same repository, but I believe it will still be necessary.